.github/workflows/main.yml

```diff
@@ -45,11 +52,11 @@ jobs:
         with:
           name: build
           path: ./build
-      - name: Deploy to Firebase
-        uses: Niweera/firebase-action@master
+    
+      - name: GitHub Action for Firebase
+        uses: w9jds/firebase-action@master
         with:
           args: deploy --only hosting
         env:
           FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
-          PROJECT_ID: ${{ secrets.REACT_APP_FIREBASE_PROJECT_ID }}
```

Comment:
sorry i missed this the last time, why did you remove the project ID?

Reply:
it is not a required field if we have already mentioned our project id in .firebaserc

Reply:
For now let's keep it there.